components/agents-catalog.tsx

```
@@ -41,6 +41,18 @@ export function AgentsCatalog({
     null,
   );
 
+  const nameLookup = useMemo(() => {
+    const map = new Map<string, string>();
+    agents.forEach((agent) => map.set(agent.id, agent.name));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)